CONTRIBUTING.md

```diff
@@ -1,5 +1,50 @@
 # Contributing to Presto
 
+Thanks for your interest in Presto.  Our goal is to build a fast, scalable and reliable distributed SQL query engine for running interactive analytic queries against data sources of all sizes ranging from gigabytes to petabytes.
+
+## Getting Started
+
+Presto's [open issues are here](https://github.com/prestodb/presto/issues). Issues that would make a good first pull request for new contributors with the `beginner-task` tag. An easy way to get started helping the project is to *file an issue*. Issues can include bugs, new features, or documentation that looks outdated. For community support, [ask for help in Slack](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM).
+
+## Contributions
+
+Presto welcomes contributions from everyone.
+
+Contributions to Presto should be made in the form of GitHub pull requests. Each pull request will
+be reviewed by a contributor or [committer](https://github.com/prestodb/presto/wiki/committers) in the project.
+Large contributions should have an associated Github issue.
+
+## Pull Request Checklist
+
+- Branch from the master branch and, if needed, rebase to the current master
+  branch before submitting your pull request. If it doesn't merge cleanly with
+  master you may be asked to rebase your changes.
+
+- Ensure that each commit is correct independently (i.e., each commit should compile and pass tests).
```

Comment:
It seems like this should delegate to the review and commit guidelines to reduce duplication?

Reply:
I think it's useful to highlight the important stuff as bullets, as `CONTRIBUTING.md` should be more easily discoverable than the wiki guidelines, and this is more focused to the contributors than committers.  How about I rework it to something like: 

```
- Make sure you follow the [review and commit guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines), in particular:

  - Ensure that each commit is correct independently (i.e., each commit should compile and pass tests).
    When possible, the size of the commit should be reduced for ease of review, while still ensuring
    that it is independently correct.

  - Make sure commit messages [follow these guidelines](https://chris.beams.io/posts/git-commit/).  In particular (from the guidelines):

    * Separate subject from body with a blank line
    * Limit the subject line to 50 characters
    * Capitalize the subject line
    * Do not end the subject line with a period
    * Use the imperative mood in the subject line
    * Wrap the body at 72 characters
    * Use the body to explain what and why vs. how
    ```

Reply:
SGTM